### DISCUSSION TITLE
How to configure Logging of IHP components, for example postgres access. Its sometimes too verbose for me.

### DISCUSSION THREAD

**Discussion by michaelfliegner** [digitallyinduced/ihp#1128] [Q&A] [answered]
How to configure Logging of IHP components, for example postgres access. Its sometimes too verbose for me.

**Comment by mpscholten** [comment, marked as answer]
Right now it's not possible to configure that.

Can you show an example where it's too verbose?

**Comment by michaelfliegner** [comment]
well  then for general use its ok. only sometimes it's a bit hard to find my own log messages. could I output mine in colour?

**Reply by jeyj0** [reply]
A quick search lead me to pretty-terminal, a haskell package to easily apply ansi-escape sequences to String/Text. That might work for your terminal when doing putStrLn-style logging (https://hackage.haskell.org/package/pretty-terminal). I haven't tried it myself I have to say though, so it might be that I missed something.
